terraform/global/um-pdp-engine/config/dependencies.tf

```diff
@@ -0,0 +1,7 @@
+resource "null_resource" "waitfor-login-service" {
```

Comment:
This file is no longer necessary. 
For module dependencies, we can use main.tf file on terraform/test/main.tf with a line stating:

module_depends_on = [module.um-login-service]